### DISCUSSION TITLE
我想用这个做服务端，实现大规模的聊天室，应该如何做呢

### DISCUSSION THREAD

**Discussion by happyworl-cyber** [Tencent/tquic#158] [General]
我想用这个做服务端，实现大规模的聊天室，应该如何做呢
可以直接联系你们吗

**Comment by happyworl-cyber** [comment]
或者用在游戏里，大概的应用思路能简单说下么，谢啦🌹

**Comment by iyangsj** [comment]
TQUIC仅是一个QUIC传输协议库。

如果你打算从零开发这些程序，可以从  [simple_server](https://tquic.net/docs/category/tutorial) 示例程序，了解如何使用TQUIC进行网络编程。
如果你要在现有聊天室服务端或游戏程序来集成TQUIC，取决于当前所使用的网络库来进行适配对接。

**Reply by happyworl-cyber** [reply]
了解了，谢谢🙏
